### ISSUE TITLE
Memory<T> does not support stack allocated memory

### ISSUE THREAD

**Issue by RobertGBryan** [dotnet/docs#25548]
[Enter feedback here]

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 4819512e-1044-1c57-0de1-447164220fd5
* Version Independent ID: d6cf8d26-fb2f-7138-f0c7-ab529a5b9cde
* Content: [Memory&lt;T&gt; and Span&lt;T&gt; usage guidelines](https://docs.microsoft.com/en-us/dotnet/standard/memory-and-spans/memory-t-usage-guidelines)
* Content Source: [docs/standard/memory-and-spans/memory-t-usage-guidelines.md](https://github.com/dotnet/docs/blob/main/docs/standard/memory-and-spans/memory-t-usage-guidelines.md)
* Product: **dotnet-fundamentals**
* GitHub Login: @tdykstra
* Microsoft Alias: **tdykstra**

**Comment by RobertGBryan** [comment]
Sorry, I had some issues with submitting this. It seemed to post it automatically for me when I was away from my computer. So, I'm going to close this one and refer you to issue 25551.

**Comment by RobertGBryan** [comment]
I just wanted to mention that there are other problems with the website here. For one, I did not receive any emails when you responded to this issue. The last issue that I submitted, I received an email notifying me that someone had responded. 

The 2nd issue is that when I tried to click on the Preview tab, it said there was an error. I thought maybe I was the cause for it because I was using backquotes to enclose a generic type, which uses angled brackets - which are not displayed properly if backquotes are not present. So, I thought maybe I forgot a matching backquote. But, then the Preview tab started to work correctly again - without me doing anything.
